pennylane_sf/gbs.py

```diff
@@ -76,10 +73,8 @@ class StrawberryFieldsGBS(StrawberryFieldsSimulator):
 
     _capabilities = {"model": "cv", "provides_jacobian": True}
 
-    def __init__(
-        self, wires, *, analytic=True, cutoff_dim, shots=1000, use_cache=False, samples=None
-    ):
-        super().__init__(wires, analytic=analytic, shots=shots)
+    def __init__(self, wires, *, cutoff_dim, shots=None, use_cache=False, samples=None):
```

Comment:
In the others we changed the order to `shots, cutoff_dim`, do we want to do the same here?

Reply:
Sure! I'll do it now.